src/CachePlugin.php

```diff
@@ -186,8 +209,11 @@ protected function isCacheable(ResponseInterface $response)
         if (!$this->config['respect_cache_headers']) {
             return true;
         }
-        if ($this->getCacheControlDirective($response, 'no-store') || $this->getCacheControlDirective($response, 'private')) {
-            return false;
+
+        foreach ($this->config['respect_response_cache_directives'] as $cacheDirective) {
+            if ($this->getCacheControlDirective($response, $cacheDirective)) {
```

Comment:
Yes you are right, I will look into a way to do this. We can't use a constant with an array value as the cacheplugin supports php >=5.4